### DISCUSSION TITLE
Backups

### DISCUSSION THREAD

**Discussion by dicastro** [hassio-addons/addon-bitwarden#93] [Q&A] [answered]
This is just a question.

When a HomeAssistant snapshot is created... the bitwarden data (passwords) is included in the snapshot? If it is not the case, how can the bitwarden data be backed up?

**Comment by frenck** [comment, marked as answer]
Yes, they are in the snapshots.

**Reply by dicastro** [reply]
Sorry for creating an issue to ask a question, I did not know the discussions functionality.

I've just installed the plugin and configured it, and I've started to explore it. I've seen a 'Backup' button in the admin page. What is this button for? Do I have to click this button before making hassio snapshot?

**Reply by frenck** [reply]
Not needed. Snapshots cover it.

**Reply by diamant-x** [reply]
Just to complete this wonderful answer, Is it required to stop the addon before backing up, [as in the MariaDB addon](https://github.com/home-assistant/addons/issues/1575#issuecomment-753531812), for the backup-restore to be correct/successful? Thanks.

**Reply by frenck** [reply]
Nope, not needed.

**Reply by mgmooij** [reply]
Does this include all vaults, also from organisations and personal passwords from all users?

**Comment by AaronDavidSchneider** [comment]
Just wanted to say that I love the ease of the backup functionality from homeassistant and that its really nice that these backups of vaultwarden are (as far as I understand) directly useable for the 'normal' vaultwarden container.
Maybe its a good idea to guide the users to a backup practice in the addon documentation? What do you think @frenck?
I would be willing to create a PR for that.

**Reply by frenck** [reply]
No sure what to cover... just create backups in HA and it is handled 🤷
